plugin/user_plugin_server.py

```
@@ -214,22 +381,702 @@ async def plugin_push_message(payload: PluginPushMessageRequest):
+        raise handle_plugin_error(e, f"Failed to reload plugin {plugin_id}", 500) from e
+
+
+# ========== 性能监控路由 ==========
+
+@app.get("/plugin/metrics")
+async def get_all_plugin_metrics(_: str = require_admin):
+    """
+    获取所有插件的性能指标
+    
+    - GET /plugin/metrics
+    - 需要管理员验证码（Bearer Token）
```

Ruff (0.14.10)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)